docs/Training-on-Amazon-Web-Service.md

```diff
@@ -1,40 +1,144 @@
 # Training on Amazon Web Service
 
-This page contains instructions for setting up an EC2 instance on Amazon Web Service for training ML-Agents environments. Visual observations is not supported currently and you need to enable headless mode.
+This page contains instructions for setting up an EC2 instance on Amazon Web Service for training ML-Agents environments. 
 
-## Recommended AMI
-You can get started with an EC2 instance with the Deep Learning AMI (Ubuntu) listed under AWS Marketplace [AMI](https://aws.amazon.com/marketplace/pp/B077GCH38C). Choose the python3 environment within that ami which gives you the python3 and CUDA 9.0 environment.
+## Preconfigured AMI
+
+We've prepared an preconfigured AMI for you with the ID: `someID` in the `us-east-1` region. It was created as a modification of [Deep Learning AMI (Ubuntu)](https://aws.amazon.com/marketplace/pp/B077GCH38C). If you want to do training without the headless mode, you need to enable X Server on it. After launching your EC2 instance using the ami and ssh into it, run the following commands to enable it:
```

Comment:
Should this say `someID`?

Reply:
The reason I put someID here is because we haven't merged develop to master yet. Once we do, I will setup the instance so that it is pointing to our master, and then make an AMI out of that, and put that AMI's ID here.